LAGraph/cmake_modules/FindGraphBLAS.cmake

```diff
@@ -83,11 +85,14 @@ if ( GraphBLAS_FOUND )
         else ( )
             add_library ( GraphBLAS::GraphBLAS ALIAS ${_graphblas_aliased} )
         endif ( )
-        if ( GRAPHBLAS_VERSION LESS "8.3.0" )
-            # workaround for SuiteSparse:GraphBLAS 8.2.x (did not have "/Include")
+        if ( GRAPHBLAS_VERSION LESS "8.3.0" AND _lagraph_gb_common_tree )
+            # workaround for incorrect INTERFACE_INCLUDE_DIRECTORIES of
+            # SuiteSparse:GraphBLAS 8.2.x before installation
+            # (did not have "/Include")
             get_property ( _inc TARGET GraphBLAS::GraphBLAS PROPERTY
                 INTERFACE_INCLUDE_DIRECTORIES )
-            include_directories ( ${_inc}/Include )
+            target_include_directories ( GraphBLAS::GraphBLAS INTERFACE
```

Comment:
I'm guessing that this should be

`target_include_directories ( ${_graphblas_aliased} INTERFACE ${_inc}/Include )
`

instead, and likewise below?